playbooks/roles/ocp-cronjob/README.md

```diff
@@ -0,0 +1,66 @@
+OCP Cronjob creation
+=======================================
+
+The role *"roles/ocp-cronjob"* provides automated creation of cronjobs. User can add cronjob value in the role variables for creation **Role Variables** section.
+This role will create the cronjob and we can monitor the jobs usig oc commands
+
+
+Requirements
+------------
+
+ - Running OCP 4.x cluster.
+
+
+Role Variables
+--------------
+
+| Variable                       | Required | Default                   |Comments                                                                                                                                                                                                                                                                          |
+|--------------------------------|----------|---------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| enable_cronjob        		 | no      | false                     | Flag to be set to true to enable cronjob |   
+| cronjob_schedule        		  	 | no      | * * * * *               | can be updated as needed |                                                                                                                                                                                                        |
+| cronjob_concurrencyPolicy           | no       | Allow              		| Value can be anything from ["Allow","forbid","replace"]     |
+| cronjob_startingDeadlineSeconds     | no       | 200              			| Can be updated as needed    |
```

Comment:
Can you describe the variables in a better way? The variable description should explain what the value does.

Reply:
Description for variables has been added.